packages/expo-updates/e2e/fixtures/project_files/maestro/tests/devClient_runUpdate.yml

```diff
@@ -14,8 +14,13 @@ onFlowStart:
     label: Tap on "Enter URL manually"
     text: Enter URL manually
 - tapOn:
-    label: Tap on "Updates URL" text field
+    label: Tap on "Updates URL" text field for iOS
+    text: "http://10.0.0.25:8081"
```

Comment:
@alanjhughes I made an adjustment to the Maestro flow here to fix the dev client E2E test. After the UI refresh, the original code doesn't work on iOS any more, so now I have different taps for iOS and Android.